main.tf

```diff
@@ -204,6 +204,10 @@ resource "aws_rds_cluster_instance" "this" {
     update = try(var.instance_timeouts.update, null)
     delete = try(var.instance_timeouts.delete, null)
   }
+
+  depends_on = [
```

Comment:
It already has a dependency https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/pull/360/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR182

This change is not necessary